File: backend/src/main/java/com/fdsa/altairis/dto/response/BookingResponse.java
package com.fdsa.altairis.dto.response;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.time.LocalDate;

@Getter
@Setter
@ToString
public class BookingResponse {
    private Long id;
    private String bookingRef;

    private Long hotelId;
    private String hotelCode;
    private String hotelName;

    private Long roomTypeId;
    private String roomTypeCode;
    private String roomTypeName;

    private LocalDate checkIn;
    private LocalDate checkOut;
    private Integer rooms;

    private String customerName;
    private String notes;

    private String status;
}

File: frontend/src/app/interfaces/response/booking.response.ts
export interface BookingResponse {
  id: number;
  bookingRef: string;

  hotelId: number;
  hotelCode: string;
  hotelName: string;

  roomTypeId: number;
  roomTypeCode: string;
  roomTypeName: string;

  checkIn: string;
  checkOut: string;
  rooms: number;

  customerName: string;
  notes?: string;

  status: string; // 'C'/'X'/'P'
}
